devel/libecl/include/ert/ecl/ecl_grid.h

```diff
@@ -172,7 +172,7 @@ extern "C" {
   void                    ecl_grid_grdecl_fprintf_kw( const ecl_grid_type * ecl_grid , const ecl_kw_type * ecl_kw , const char * special_header , FILE * stream , double double_default);
   bool                    ecl_grid_test_lgr_consistency( const ecl_grid_type * ecl_grid );
 
-  void                    ecl_grid_fwrite_EGRID(  ecl_grid_type * grid , const char * filename);
+  void                    ecl_grid_fwrite_EGRID(  ecl_grid_type * grid , const char * filename, bool is_metric);
```

Comment:
The variable should not be called `is_metric`; the grid itself *is* metric (or not), the input parameter here is the format you want to use for the output and a sensible name would be `metric_output` or `write_metric`.